src/Application.php

```diff
@@ -624,11 +631,11 @@ public function runErrorController(string $body, int $httpCode = 400): Response
           # Run Controller's handler
           return $routeHandler->$handlerMethod([]);
         }
-        \logger()->error('Failed to create error controller',[
+        \logger()?->error('Failed to create error controller',[
```

Comment:
we could just fix the type hint, if there's no log, there'll be no app running anyways